### DISCUSSION TITLE
Debian Bullseye Packages #656

### DISCUSSION THREAD

**Discussion by wouterh-dev** [OpenIDC/mod_auth_openidc#693] [Q&A]
Good evening,
On the 14th Debian Bullseye has been released as new stable.

Are there any plans to release the debs built for it as well?

Thank you.

https://github.com/zmartzone/mod_auth_openidc/issues/656

**Comment by zandbelt** [comment]
no concrete plans yet but I believe 2.4.9, which is relatively new, is included in the distribution channel itself

**Comment by chrisinmtown** [comment]
Yes I see mod-auth-openidc v2.4.9-1 listed as a Debian-Bullseye package here:
https://packages.debian.org/bullseye/libapache2-mod-auth-openidc
